llvm/lib/SYCLLowerIR/ESIMD/LowerESIMD.cpp

```diff
@@ -653,8 +653,6 @@ class ESIMDIntrinDescTable {
          {"__spirv_ConvertFToBF16INTEL", {a(0)}}},
         {"__devicelib_ConvertBF16ToFINTEL",
          {"__spirv_ConvertBF16ToFINTEL", {a(0)}}},
-        {"addc", {"addc", {l(0)}}},
```

Comment:
let's keep the lowering in LowerESIMD.cpp for a while - see the reasoning in other PR: https://github.com/intel/llvm/pull/12935